docs/manuals/admin/timescale.md

```diff
@@ -0,0 +1,187 @@
+# Timescale
+
+[Timescale][timescale] is one of the time series databases that can be
+used with QuantumLeap as a back end to store NGSI entity time series.
+Indeed, QuantumLeap provides full support for storing NGSI entities in
+Timescale, including geographical features (encoded as GeoJSON or NGSI
+Simple Location Format), structured types and arrays. Moreover, it is
+possible to dynamically select, at runtime, which storage back end to
+use (Crate or Timescale) depending on the tenant who owns the entity
```

Comment:
i would not cover discussion for multiple support here. better in index or some other generic place https://github.com/smartsdk/ngsi-timeseries-api/blob/master/docs/manuals/index.md 
index should be updated as well

Reply:
moved content to landing page, see 05058d3 and 0b8bab4